package.json

```diff
@@ -460,6 +460,12 @@
   },
   "packageManager": "yarn@4.9.2",
   "dependenciesMeta": {
+    "cypress": {
+      "built": true
+    },
+    "msw": {
+      "built": true
+    },
     "nx": {
```

Comment:
we can remove this now we're disabling all postinstall scripts globally, right? 🤔